content/modules/ROOT/pages/index.adoc

```diff
@@ -353,13 +351,13 @@ tree taxonomy/  | head
 taxonomy/
 ├── CODE_OF_CONDUCT.md
 ├── compositional_skills
-│   ├── extraction
-│   │   ├── abstractive
-│   │   │   ├── abstract
-│   │   │   │   └── qna.yaml
-│   │   │   ├── key_points
-│   │   │   │   └── qna.yaml
-│   │   │   ├── main_takeaway
+│   ├── arts
+│   ├── engineering
+│   ├── geography
+│   ├── grounded
+│   │   ├── arts
+│   │   ├── engineering
+│   │   ├── geography
```

Comment:
Hmm so I think the previous example skipped to a full directory to show a full qna.yaml (and I guess attributes) directory structure. I think it would also be nice to show here?